pkg/deployer/testing/integration_test_helper.go

```diff
@@ -0,0 +1,1155 @@
+//go:build integration
+// +build integration
+
```

Comment:
Not really necessary to have tags in this file, since the test entrypoints are elsewhere, but it doesn't hurt anything other; just confuses some IDEs.